tedana/workflows/tedana.py

```diff
@@ -171,17 +192,30 @@ def _get_parser():
             "Value used for random initialization of ICA "
             "algorithm. Set to an integer value for "
             "reproducible ICA results. Set to -1 for "
-            "varying results across ICA calls. "
+            "varying results across ICA calls. This"
+            "applies to both fastica and robustica methods."
+        ),
+        default=DEFAULT_SEED,
+    )
+    optional.add_argument(
+        "--n_robust_runs",
+        dest="n_robust_runs",
+        type=int,
+        help=(
+            "The number of times robustica will run."
```

Comment:
```suggestion
            "The number of times robustica will run. "
```
(noticed a missing space)